src/renderer/nav/Validator.tsx

```diff
@@ -89,6 +89,15 @@ const Validator = () => {
     });
   }, 222);
 
+  useEffect(() => {
+    if (validator.net === Net.Localhost) {
+      window?.promiseIpc.send('DOCKER-CheckDocker').catch((err) => {
+        logger.error(err);
+        toast.error(err.message);
```

Comment:
I'm inclined to use the gentler `toast.warning` here, since it's not something the user or app did wrong, but just a heads up. Let's do that for now.

Reply:
yeah. sounds better